frontend/src/app/main/ui/dashboard/grid.cljs

```diff
@@ -56,15 +59,22 @@
 
 (defn render-thumbnail
   [file-id revn]
-  (->> (mw/ask! {:cmd :thumbnails/generate-for-file
-                 :revn revn
-                 :file-id file-id})
-       (rx/mapcat (fn [{:keys [fonts] :as result}]
-                    (->> (fonts/render-font-styles fonts)
-                         (rx/map (fn [styles]
-                                   (assoc result
-                                          :styles styles
-                                          :width 252))))))))
+  (if (features/active-feature? @st/state "plugins/runtime")
```

Comment:
this should be "render-wasm/v1" i think

Reply:
ups! yes, I failed with the copy/paste lol